datasource-remote/src/main/java/cmm/apps/esmorga/datasource_remote/user/model/UserRemoteModel.kt

```diff
@@ -0,0 +1,13 @@
+package cmm.apps.esmorga.datasource_remote.user.model
+
+data class UserRemoteModel(
+    val accessToken: String,
```

Comment:
do you store the tokens in local?

Reply:
not yet, we'll need it for sure, but we'll need to think where and how we'll do that